### PR TITLE
[E2E Tests] fix: set versions ony for PRs

### PR DESCRIPTION
**Pull request by mdelapenya**
We want to use default versions per branch when running after a merge

## What does this PR do?
It creates the parameters for the E2E tests build dynamically, which means: if it's a PR, it will use the snapshots created by the PR.

## Why is it important?
It will use the default values for merge commits on Beats repo, triggering the job for the proper maintenance branch

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Related issues

- Relates #21100

### CODE REVIEW

File: .ci/packaging.groovy [hunk 1]
```diff
@@ -245,19 +245,25 @@ def triggerE2ETests(String suite) {
...
   def e2eTestsPipeline = "e2e-tests/e2e-testing-mbp/${branchName}"
+
+  def parameters = [
+    booleanParam(name: 'forceSkipGitChecks', value: true),
+    booleanParam(name: 'forceSkipPresubmit', value: true),
+    booleanParam(name: 'notifyOnGreenBuilds', value: !isPR())
+    string(name: 'runTestsSuites', value: suite),
+    string(name: 'GITHUB_CHECK_NAME', value: env.GITHUB_CHECK_E2E_TESTS_NAME),
+    string(name: 'GITHUB_CHECK_REPO', value: env.REPO),
+    string(name: 'GITHUB_CHECK_SHA1', value: env.GIT_BASE_COMMIT),
```
Comment (mdelapenya) [inline .ci/packaging.groovy:256]: I'm thinking that these three should be added only for PRs, thoughts?

Comment (v1v) [reply .ci/packaging.groovy:256]: Uhm, if running those builds are PR agnostic, then the GH check will be reported even for branches and tags independently. I see value by having them, unless it's already triggered as part of another pipeline

File: .ci/packaging.groovy [hunk 2]
```diff
@@ -245,19 +245,25 @@ def triggerE2ETests(String suite) {
 
   def branchName = isPR() ? "${env.CHANGE_TARGET}" : "${env.JOB_BASE_NAME}"
   def e2eTestsPipeline = "e2e-tests/e2e-testing-mbp/${branchName}"
+
+  def parameters = [
+    booleanParam(name: 'forceSkipGitChecks', value: true),
+    booleanParam(name: 'forceSkipPresubmit', value: true),
+    booleanParam(name: 'notifyOnGreenBuilds', value: !isPR())
```
Comment (v1v) [inline .ci/packaging.groovy:252]:
```suggestion
    booleanParam(name: 'notifyOnGreenBuilds', value: !isPR()),
```

File: .ci/packaging.groovy (hunk 1 above)
Comment (mdelapenya) [reply .ci/packaging.groovy:256]: Ok then, let's maintain them, as they could help contributing the e2e tests check on a merge commit. Thank you!

**Review by cachedout — APPROVED** [review]

**Pull request merged**